### DISCUSSION TITLE
Review

### DISCUSSION THREAD

**Discussion by BrianMunene96** [HYF-Class19/workflow-group1#12] [General]
## Group 1 discussion forum.

Hi team, we can use this section also to ask questions or receive guidance.

**Question:** How's your progress so far?

**Comment by sammou00** [comment]
Keep up the good work, just remove PR from the project board and only add issues as we agreed in the class.

**Reply by BrianMunene96** [reply]
Noted. Thank you 😊

**Reply by BrianMunene96** [reply]
> Keep up the good work, just remove PR from the project board and only add issues as we agreed in the class.

@filmon-goitom

**Reply by sammou00** [reply]
Hello, I have noticed , that you don't have a project board ?

**Reply by BrianMunene96** [reply]
We had a project board but it seems it was closed maybe by mistake. I'll just create another one an we can put our issues in the correct section.

**Comment by sammou00** [comment]
Hello, Good job, I think adding your images  to your md files , will be better, for example  john.md , we can add john.img in his md file instead of README.md

**Reply by BrianMunene96** [reply]
Thank you for your feedback. Some of our members have pictures in their MD file as well. Maybe it's okay to have images in both files?

**Reply by sammou00** [reply]
@BrianMunene96  it is up to you and your team, there is no right or wrong.

**Comment by sammou00** [comment]
If your colleagues done reviewing your issues, please move them to `done`.

**Reply by sammou00** [reply]
Normally , in the project board we have  
`To Do` : for all the issues need to be done.
 `Doing` :  for the current issue you are working on.
`Ready For Review` : so your colleagues know your issue(s) is/ are ready for review 
`Need A Revision` :  in case , your colleagues found something can be improved after reviewing your issue.
`Done` : when you are done.

**Reply by BrianMunene96** [reply]
Okay, I'll add another section **(needs revision)** and we can move the issues to the appropriate column
